File: app/src/main/java/com/dwayne/com/showcase/MainActivity.java
package com.dwayne.com.showcase;

import android.content.Intent;
import android.view.View;

import com.dwayne.com.showcase.focustest.FocusTestActivity;
import com.dwayne.com.showcase.textlink.TextLinkActivity;

import butterknife.OnClick;

/**
 * @author Dwayne
 * @email dev1024@foxmail.com
 * @time 20/4/25 23:10
 * @change
 * @chang time
 * @class 演示代码
 */


public class MainActivity extends BaseActivity {

    @Override
    public int intiLayout() {
        return R.layout.activity_main;
    }

    @Override
    public void initView() {
        initToolBar("博客演示代码", false);
    }

    @Override
    public void initData() {

    }

    @OnClick({R.id.tv_text_link, R.id.tv_focus_test})
    public void onViewClicked(View view) {
        switch(view.getId()) {
            case R.id.tv_text_link:
                startActivity(new Intent(this, TextLinkActivity.class));
                break;
            case R.id.tv_focus_test:
                startActivity(new Intent(this, FocusTestActivity.class));
                break;
        }
    }
}

File: app/src/main/java/com/dwayne/com/showcase/MainActivity.kt
package com.dwayne.com.showcase

import android.content.Intent
import android.view.View
import android.widget.TextView
import butterknife.OnClick
import com.dwayne.com.showcase.ui.FocusTestActivity
import com.dwayne.com.showcase.ui.PathMeasureActivity
import com.dwayne.com.showcase.ui.TextLinkActivity

/**
 * @author Dwayne
 * @email dev1024@foxmail.com
 * @time 20/4/25 23:10
 * @change
 * @chang time
 * @class 演示代码
 */
class MainActivity : BaseActivity() {
    override fun intiLayout(): Int {
        return R.layout.activity_main
    }

    override fun initView() {
        initToolBar("博客演示代码", false)
//        findViewById<TextView>(R.id.tv_path_measure).setOnClickListener{
//            startActivity(Intent(this, PathMeasureActivity::class.java))
//        }
    }

    override fun initData() {}

    @OnClick(R.id.tv_text_link, R.id.tv_focus_test, R.id.tv_path_measure)
    fun onViewClicked(view: View) {
        when (view.id) {
            R.id.tv_text_link -> startActivity(Intent(this, TextLinkActivity::class.java))
            R.id.tv_focus_test -> startActivity(Intent(this, FocusTestActivity::class.java))
            R.id.tv_path_measure -> startActivity(Intent(this, PathMeasureActivity::class.java))
        }
    }
}